shared-actions/npm-setup/action.yml

```diff
@@ -0,0 +1,31 @@
+name: JS/TS environment setup
+description: 'Set up node & npm config, Get repo and install its dependencies (using cache)'
+inputs:
```

Comment:
There are some teams that have had problems with the cache before.

I'd like to add an input to be able _disable_ the cache. Enabled by default, but if they want, they can specify it. Something like:
```yml
  cache:
     description: 'Use cache'
     default: true
```